static/js/consumers.js

```diff
@@ -0,0 +1,54 @@
+import * as Table from './table.js'
+import * as HTTP from './http.js'
+import * as DOM from './dom.js'
+
+const vhost = window.sessionStorage.getItem('vhost')
+let url = 'api/consumers'
+if (vhost && vhost !== '_all') {
+  url = `api/consumers/${encodeURIComponent(vhost)}`
+}
+
+const tableOptions = {
+  url,
+  keyColumns: ['channel_details'],
```

Comment:
Why wouldn't `channel_details` work as key? https://github.com/cloudamqp/lavinmq/blob/fd31b05c7d8193dccfebd48050d4c429fa20a0b6/static/js/table.js#L100

But `consumer_tag` should be added, right? 
```suggestion
  keyColumns: ['channel_details', 'consumer_tag`],
```

Reply:
I fixed so that objects (and numbers) can be used a key columns in this PR: https://github.com/cloudamqp/lavinmq/pull/529
but the consumer_tag should still be added.